clang/include/clang/Basic/SourceManager.h

```diff
@@ -489,6 +490,8 @@ class SLocEntry {
 
   bool isExpansion() const { return IsExpansion; }
   bool isFile() const { return !isExpansion(); }
+  [[nodiscard]] bool isLoaded() const { return Loaded; }
```

Comment:
Do we need to add `[[nodiscard]]` to the interfaces in this PR? I'd remove them.

Reply:
No, I guess not. It's just my clang-tidy integration being overzealous, I guess.

```suggestion
  bool isLoaded() const { return Loaded; }
```